sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -989,6 +989,46 @@ describe("LocalWorkspace", () => {
         const destroyRes = await stack.destroy({ userAgent, refresh });
         assert.match(destroyRes.stdout, /refreshing/);
     });
+    it(`operations accept configFile option`, async () => {
+        // Setup test data
+        const configPath = upath.joinSafe(__dirname, "data", "yaml", "Pulumi.dev.yaml");
+
+        // Create options objects with configFile
+        const previewOpts = { configFile: configPath };
+        const upOpts = { configFile: configPath };
+        const refreshOpts = { configFile: configPath };
+        const destroyOpts = { configFile: configPath };
+
+        // Simple verification that the options properties exist and are accepted
+        assert.strictEqual(previewOpts.configFile, configPath, "preview options should accept configFile");
+        assert.strictEqual(upOpts.configFile, configPath, "up options should accept configFile");
+        assert.strictEqual(refreshOpts.configFile, configPath, "refresh options should accept configFile");
+        assert.strictEqual(destroyOpts.configFile, configPath, "destroy options should accept configFile");
```

Comment:
This can and should still be dropped.  We're just checking that a typescript map is set up correctly. So we're just testing a language feature that's trivially correct.  It's more confusing than anything to have this in an integration test.